### PR TITLE
Basic custom favicon support

### PR DESCRIPTION
**Pull request by toortin**
Add basic custom favicon support following the model for the logo.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -7,7 +7,11 @@
 
         <title>{{ block "title" . }}Orange Forum{{ end }}</title>
 
+        {{ if .Domain.Icon }}
+        <link rel="shortcut icon" href="{{ .Domain.Icon }}?v=223">
```
Comment (s-gv) [inline templates/base.html:11]: The `?v=223` should be removed from here.

Comment (s-gv) [reply templates/base.html:11]: Also, in the Domain struct in models/domains.go, the type of Icon should be changed to `template.URL` (just like logo).

Comment (s-gv) [reply templates/base.html:11]: And can you also rebase all the commits to just one "Support per-domain favicon" commit and force push? Thx

**Comment by toortin** [comment]
I think it should be good now. Apologies, I'm new to git and don't have a lot of experience with programming yet.

**Comment by s-gv** [comment]
No problem. Thanks for the PR!

**Comment by s-gv** [comment]
Can you please add yourself to the AUTHORS file? Thx.

**Comment by toortin** [comment]
Sure, should it be its own PR?

**Comment by s-gv** [comment]
A separate commit would be great. It’s fine for it to be a separate PR.

**Comment by toortin** [comment]
I have added the commit. Do tell me if something isn't right and thanks for the patience :)

**Comment by s-gv** [comment]
Hmm. I think you need to create a new PR for it. The commit isn't showing up in the closed PR.

**Pull request merged**